guide/inheritance-wallet/backup.md

```diff
@@ -0,0 +1,279 @@
+
+As we have covered on the wallet creation page, both of the inheritance routes for Christina and David are available [after six months]({{ '/guide/inheritance-wallet/wallet-creation/#creating-the-inheritance-key-set' | relative_url }}) of wallet inactivity. 
+
+## Backup material distribution
+
+The diagram below provides a high-level overview of how Alice and Bob distribute the backup material to achieve this goal. It supports the recovery scenarios outlined above, while maintaining financial privacy for Alice and Bob during their lifetimes.
+
+{% include picture.html
+   image = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution.png"
+   retina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution@2x.png"
+   modalImage = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution@2x.png"
+   mobile = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-mobile.png"
+   mobileRetina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-mobile@2x.png"
+   modalImageMobile = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-mobile@2x.png"
+   alt-text = "A schematic illustration showing how the backup material is distributed between the parents, their children and the lawyer."
+   width = 1600
+   height = 800
+   modalWidth = 3324
+   modalHeight = 1850
+   modalWidthMobile = 1156
+   modalHeightMobile = 2936
+%}
+
+
+### Password Manager
+
+##### Alice and Bob's personal vaults
+
+Alice and Bob each store an encrypted version of the recovery kit in their own personal vaults of the password manager they use. Encrypting the recovery kit protects it from a potential security breach on the side of the password manager. 
+
+{% include picture.html
+   image = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-pwm.png"
+   retina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-pwm@2x.png"
+   mobile = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-pwm-mobile.png"
+   mobileRetina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-pwm-mobile@2x.png"
+   alt-text = "A diagram showing the sequence of recovery scenarios."
+   width = 1600
+   height = 800
+%}
+
+##### Shared family vault
+
+To make sure that Christina and David will be able to retrieve both the recovery kit  and the password to decrypt it, Alice and Bob store the following pieces of information in a shared family vault:
+
+- House safe PIN
+- Lawyer's contact details
+- Bank deposit box details
+
+### Home safe
+
+Bob places the following items in a tamper-evident bag and stores it in their home safe:
```

Comment:
Would this be the "vault" referred to above?

Reply:
It's a bit tricky, because the terminology is quite similar for both concepts. But one is physical and the other is digital: 

- the home safe is the physical safe that the Joneses have in their home
-  vaults are digital vaults inside of the password manager application. 

I've tried to visualize this in the "backup material distribution" section and diagram.